.github/workflows/build_docker_image.yml

```diff
@@ -0,0 +1,20 @@
+name: Build Docker image
+
+on:
+  push:
+    branches: ['main']
+  pull_request:
+    branches: ['main']
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check out repository
+      uses: actions/checkout@v3
+
+    - name: Build the Docker image
+      run: docker build . --file Dockerfile --tag tractolearn-docker:$(date +%Y-%m-%d-%H-%M)
```

Comment:
The image name may need to be re-worked as it will contain a colon, which may be undesirable, e.g. `tractolearn-docker:2023-02-06-08-43`, and maybe docker images usually contain an extension (`.img`).

Reply:
OK, I read the Docker documentation and the name matches the recommended standard. Resolving.